x/posts/internal/types/models/post_response_test.go

```diff
@@ -167,21 +167,19 @@ func TestPostQueryResponse_String(t *testing.T) {
 		models.NewUserAnswers(
 			models.NewUserAnswer([]models.AnswerID{models.AnswerID(1)}, liker),
 		),
-		[]models.ReactionQueryResponse{
-			models.NewReactionQueryResponse("https://example.com/like", ":like:", liker),
-			models.NewReactionQueryResponse("👍", ":+1:", otherLiker),
+		[]models.PostReaction{
+			models.NewPostReaction("https://example.com/like", ":like:", liker),
+			models.NewPostReaction("👍", ":+1:", otherLiker),
```

Comment:
The `NewPostReaction` method is wrongly used here. The first parameter is the shortcode, and the second one is the value, not the other way around. Please fix it

```suggestion
			models.NewPostReaction(":like:", "https://example.com/like", liker),
			models.NewPostReaction(":+1:", "👍", otherLiker),
```